sunscreen_compiler_common/src/lookup.rs

```diff
@@ -125,7 +161,6 @@ impl GroupLookup {
     pub fn new() -> Self {
```

Comment:
Add a simple method here

```rust
pub fn children_of(&self, id: u64) -> Vec<u64> {
  self.id_data.lookup.values().iter().map(|x| x.parent == id).collect()
}
```

which you can use to figure out the children of this group.